docs/getting-started-guides/rkt/index.md

```diff
@@ -2,108 +2,98 @@
-        "gateway": "10.255.228.1"
-    },
-    "routes": [
-      { "dst": "0.0.0.0/0" }
-    ]
+    "type": "flannel",
+    "delegate": {
+        "isDefaultGateway": true
+    }
 }
 EOF
 ```
 
-This example creates a `bridge` plugin configuration for the CNI network, specifying
-the bridge name `cbr0`. It also specifies the CIDR, in the `ipam` field.
+For more information on flannel configuration, please read [CNI/flannel README](https://github.com/containernetworking/cni/blob/master/Documentation/flannel.md).
+
+#### On GCE
 
-Creating these files for any moderately-sized cluster is at best inconvenient.
-Work is in progress to
-[enable Kubernetes to use the CNI by default]
-(https://github.com/kubernetes/kubernetes/pull/18795/files).
-As that work matures, such manual CNI config munging will become unnecessary
-for primary use cases. For early adopters, an initial example shows one way to
-[automatically generate these CNI configurations]
-(https://gist.github.com/yifan-gu/fbb911db83d785915543)
-for rkt.
+Each VM on GCE has an additional 256 IP addresses routed to it, so it is possible to forego flannel in smaller clusters.
+This can most easily be done by using the builtin kubenet plugin.
+This can be done by setting `--network-plugin=kubenet`.
 
 ### Local cluster
 
 To use rkt as the container runtime, we need to supply the following flags to kubelet:
 
-- `--container-runtime=rkt` chooses the container runtime to use. Possible values: 'docker', 'rkt'. Default: 'docker'.
-- `--rkt-path=$PATH_TO_RKT_BINARY` sets the path of rkt binary. Leave empty to use the first rkt in $PATH.
-- `--rkt-stage1-image` sets the path of the stage1 image. Local paths and http/https URLs are supported. Leave empty to use the 'stage1.aci' that locates in the same directory as the rkt binary.
+- `--container-runtime=rkt` chooses the container runtime to use.
+- `--rkt-api-endpoint=HOST:PORT` sets the endpoint of the rkt API service.
+  Leave empty to use the default one (`localhost:15441`).
+- `--rkt-path=$PATH_TO_RKT_BINARY` sets the path of rkt binary.
+  Leave empty to use the first rkt in $PATH.
+- `--rkt-stage1-image` sets the name of the stage1 image, e.g. coreos.com/rkt/stage1-coreos.
+  Leave empty to use the default stage1 image in the rkt's configuration.
 
-If you are using the [hack/local-up-cluster.sh](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/hack/local-up-cluster.sh) script to launch the local cluster, then you can edit the environment variable `CONTAINER_RUNTIME`, `RKT_PATH` and `RKT_STAGE1_IMAGE` to
-set these flags:
+If you are using the [hack/local-up-cluster.sh](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/hack/local-up-cluster.sh) script to launch the local cluster, then you can edit the environment variable `CONTAINER_RUNTIME`, `RKT_PATH` and `RKT_STAGE1_IMAGE` to set these flags:
 
 ```shell
 $ export CONTAINER_RUNTIME=rkt
 $ export RKT_PATH=$PATH_TO_RKT_BINARY
```

Comment:
Maybe note that these are optional if you have rkt configured appropriately and in your path?